hikari/api/rest.py

```diff
@@ -7736,6 +8007,54 @@ async def edit_interaction_response(
             If an internal error occurs on Discord while handling the request.
         """
 
+    @abc.abstractmethod
+    async def edit_interaction_voice_message_response(
+        self,
+        application: snowflakes.SnowflakeishOr[guilds.PartialApplication],
+        token: str,
+        attachment: typing.Union[files.Resourceish, messages_.Attachment],
+        waveform: str,
+        duration: float,
+    ) -> messages_.Message:
+        """Edit the initial response to a voice message.
+
+        Parameters
+        ----------
+        application
+            Object or ID of the application to edit a command response for.
+        token
+            The interaction's token.
+        attachment
+            If provided, the attachment to set on the message. If
+            [`hikari.undefined.UNDEFINED`][], the previous attachment, if
+            present, is not changed. If this is [`None`][], then the
+            attachment is removed, if present. Otherwise, the new attachment
+            that was provided will be attached.
```

Comment:
Not correct, as there must not be a previous attachment.

I believe that otherwise it will break (needs testing tho)